From 45e6ceff04b72dc9284be9466c4b11f2a737870f Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 12:39:08 +0800
Subject: [PATCH] Test case1 for Change file guid of INF file 

Description: Change file guid of INF file (to "9483ef8b-9a36-32f8-4ca5-5b66e45370ec") 
in TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.inf.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.inf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.inf b/TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.inf
index aaf6858291..b08caa6fa4 100644
--- a/TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.inf
+++ b/TestPkg/Library/BaseFakeLibraryLib/BaseFakeLibraryLib.inf
@@ -15,7 +15,7 @@
   INF_VERSION                    = 0x00010005
   BASE_NAME                      = BaseFakeLibraryLib
   MODULE_UNI_FILE                = BaseFakeLibraryLib.uni
-  FILE_GUID                      = 9172fe8b-9a36-40f8-bba5-5d48e45370ec
+  FILE_GUID                      = 9483ef8b-9a36-32f8-4ca5-5b66e45370ec
   MODULE_TYPE                    = BASE
   VERSION_STRING                 = 1.0
   LIBRARY_CLASS                  = FakeLibraryLib
-- 
2.27.0.windows.1

